resources/public/api/conf/1.0/createOrAmendCodingOutUnderpayments.yaml

```diff
@@ -3,21 +3,11 @@
     This endpoint enables you to overwrite coding out underpayments and debt amounts held by HMRC for a previous tax year and given National Insurance number, and to amend previously submitted overwrite amounts. This endpoint can only be used after the tax year has ended.
```

Comment:
I know it wasn't done in this PR but could be worth updating while we are here: It seems all the files are not named correctly. Usually, we go by endpoint group followed by the method in snake case i.e. createOrAmendCodingOutUnderpayments -> coding_out_underpayments_create_or_amend.yaml